database_scripts/tabels/create_category.sql

```diff
@@ -1,9 +1,9 @@
 use customer_db
 
-create table Category
+CREATE TABLE [dbo].[Categories] 
 (
-ID int PRIMARY KEY,
-Name nvarchar(100),
-[Description] nvarchar(100)
+    [ID] [int] NOT NULL IDENTITY,
+    [Name] [nvarchar](max),
+    [Description] [nvarchar](max),
+    CONSTRAINT [PK_dbo.Categories] PRIMARY KEY ([ID])
 )
-
```

Comment:
Tu jest ok, tylko musisz dodać NOT NULL przy polach, które są IsRequired() i długości nvarcharów np. [Name] [nvarchar](100) oraz zmień `use customer_db` na `use BProject_db`